frontend/packages/core/src/WorkflowLayout/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import type { CSSObject, Theme } from "@mui/material";
+
+import styled from "../styled";
+import { Typography } from "../typography";
+
+export type LayoutVariant = "standard" | "wizard" | "custom";
+
+export type LayoutProps = {
+  variant: LayoutVariant;
+  heading?: string | React.ReactElement;
+  showHeader?: boolean;
+};
+
+type StyledVariantComponentProps = {
+  theme: Theme;
+  $variant: LayoutVariant;
+};
+
+const BASE_CONTAINER_STYLES: CSSObject = {
+  display: "flex",
+  flexDirection: "column",
+  width: "100%",
+  overflowY: "auto",
+};
+
+const getContainerVariantStyles = (variant: LayoutVariant, theme: Theme) => {
+  const layoutVariantStylesMap: { [key in LayoutVariant]: CSSObject } = {
+    standard: {
+      ...BASE_CONTAINER_STYLES,
+      padding: theme.spacing(theme.clutch.spacing.md),
+    },
+    wizard: {
+      ...BASE_CONTAINER_STYLES,
+      width: "800px", // Taken from the Wizard Component default width
+      padding: theme.spacing(theme.clutch.spacing.lg, theme.clutch.spacing.none),
+      margin: theme.spacing(theme.clutch.spacing.none, "auto"),
+    },
+    // No styles
+    custom: {},
+  };
+  return layoutVariantStylesMap[variant];
+};
+
+const LayoutContainer = styled("div")(
+  ({ $variant, theme }: StyledVariantComponentProps) =>
+    getContainerVariantStyles($variant, theme) as any
+);
+
+const PageHeader = styled("div")(({ $variant, theme }: StyledVariantComponentProps) => ({
+  padding: theme.spacing(
+    theme.clutch.spacing.base,
+    $variant === "wizard" ? theme.clutch.spacing.md : theme.clutch.spacing.none
+  ),
+  width: "100%",
+}));
+
+const HeaderTitle = styled(Typography)({
+  lineHeight: 1,
+});
+
+const WorkflowLayout = ({
+  variant,
+  showHeader,
+  heading,
+  children,
+}: React.PropsWithChildren<LayoutProps>) => {
+  return (
+    <LayoutContainer $variant={variant}>
+      {showHeader !== false && (
```

Comment:
Wondering about this condition, seems funny checking for` !== false`, wouldn't it be the same as just checking for `showHeader = true`?, or maybe a prop name change to `hideHeader` if the default is to show it?

Reply:
My reasoning is that it won't be used unless setting it to `false`. 

I could default it as `showHeader = true` in the props destructuring and doing the check to `=== true`, though.

Reply:
🤔 my concern is the prop name `showHeader`, and that it will only take effect if set to `false`, kinda goes against what `show` conveys. Or maybe I'm getting it wrong, double negatives confuse me 😆

Reply:
Ahh, I see now. I think that makes sense, I'll submit a change for it.

Reply:
Done in [275ea93](https://github.com/lyft/clutch/pull/3143/commits/275ea93fb34028b3978fd5888e2b308de56e1011)